pyproject.toml

```diff
@@ -1,3 +1,45 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "heir_py"
+version = "0.0.1"
+authors = [
+    { name = "Jeremy Kun", email = "jkun@google.com" },
+    { name = "Asra Ali", email = "asraa@google.com" },
+    { name = "Alexander Viand", email = "alexander.viand@intel.com" },
+]
+description = "Python frontend for the HEIR compiler"
```

Comment:
Is it really correct to describe this package as "just" a frontend for the HEIR compiler when it actually ships with the compiler + backend(s)?

Reply:
Updated